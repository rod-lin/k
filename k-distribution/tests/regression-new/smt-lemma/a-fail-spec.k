// Copyright (c) 2020 K Team. All Rights Reserved.

module A-FAIL-SPEC

imports A

rule <k> zoo(foo(X)) => koo(bar(X)) ... </k>
    requires X <Int 0

endmodule
