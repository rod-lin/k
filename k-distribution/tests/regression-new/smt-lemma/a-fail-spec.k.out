<k>
  koo ( foo ( V0 ) ) ~> V1
</k>
